src/chart/pie/pieLayout.js

```diff
@@ -80,19 +80,7 @@ export default function (seriesType, ecModel, api, payload) {
         data.each(valueDim, function (value, idx) {
             var angle;
             if (isNaN(value)) {
-                data.setItemLayout(idx, {
-                    angle: NaN,
-                    startAngle: NaN,
-                    endAngle: NaN,
-                    clockwise: clockwise,
-                    cx: cx,
-                    cy: cy,
-                    r0: r0,
-                    r: roseType
-                        ? NaN
-                        : r
-                });
-                return;
+                value = 0;
```

Comment:
@pissang mentioned that if set value as `0`, border still exists, which is not expected.